python/pyarrow/_fs.pyx

```diff
@@ -40,7 +40,9 @@ cdef inline c_string _path_as_bytes(path) except *:
 
 
 cdef class FileStats:
-    """FileSystem entry stats"""
+    """
+    FileSystem entry stats.
```

Comment:
This type of change is pointless. Can't you disable this rule in Numpydoc?

Reply:
I have actually disabled it, because cython's embedsignature directive prepends the docstring with the signature, which always violates the numpydoc rule.

I also tried to disable the embedsignature, but it is useful for identifying the object from the CLI output.

BTW I'm not sure what's the right policy on this, because numpydoc rule explicitly states that:
> GL01: Docstring text (summary) should start in the line immediately after the opening quotes (not in the same line, or leaving a blank line in between)

But the examples from the [guide](https://numpydoc.readthedocs.io/en/latest/format.html) violates that :)

cc @jorisvandenbossche

Reply:
> But the examples from the guide violates that :)

See https://github.com/numpy/numpydoc/issues/241

Historic background: the validation rules were kind of blindly copied from pandas to numpydoc, and thus there are indeed some rules that numpydoc itself doesn't necessarily follow (because in pandas we decided to make a certain choice where the numpydoc or PEP257 is ambiguous or allows different options).

For that reason I also mentioned before we should probably pick a few rules that we think are important (like validating that all parameters have a description).